### PR TITLE
Add incrementallyLoadTextures to 3D Tiles

### PR DESCRIPTION
**Pull request by JudyWeng**
For #4345

### CODE REVIEW

File: Source/Scene/ModelInstanceCollection.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ define([
      * @param {Boolean} [options.asynchronous=true] Determines if model WebGL resource creation will be spread out over several frames or block until completion once all glTF files are loaded.
      * @param {Boolean} [options.debugShowBoundingVolume=false] For debugging only. Draws the bounding sphere for the collection.
      * @param {Boolean} [options.debugWireframe=false] For debugging only. Draws the instances in wireframe.
+     * @param {Boolean} [options.incrementallyLoadTextures=true] Determine if textures may continue to stream in after the model is loaded.
```
Comment (lilleyse) [inline Source/Scene/ModelInstanceCollection.js]: Minor organizational thing that I didn't catch earlier - can you put this line above the debug ones?

**Comment by lilleyse** [comment]
Thanks @JudyWeng for your first Cesium pull request! As part of this PR can you add yourself to `CONTRIBUTORS.md`?

File: Source/Scene/ModelInstanceCollection.js (hunk 1 above)
Comment (JudyWeng) [inline Source/Scene/ModelInstanceCollection.js]: Hey Sean, so I did make this change locally while I'm in the incrementallyLoadTextures branch. Would I just commit and push this change like last time?

**Comment by lilleyse** [comment]
Looks great!

**Comment by pjcozzi** [comment]
@JudyWeng congrats on your first pull request into Cesium.  Thanks for the contribution!

**Pull request merged**